syntax = "proto2";

package opentxs.proto;
option optimize_for = LITE_RUNTIME;

import public "ContactEnums.proto";

message ContactItem {
    optional uint32 version = 1;
    optional ContactItemType type = 2;
    optional string value = 3;
    optional sint64 start = 4;
    optional sint64 end = 5;
    repeated ContactItemAttribute attribute = 6;
}
